packages/frontend-2/components/auth/RegisterWithEmailBlock.vue

```diff
@@ -43,6 +43,23 @@
       :class="`mt-2 overflow-hidden ${pwdFocused ? 'h-8' : 'h-0'} transition-[height]`"
     />
     <FormButton submit full-width class="mt-4" :disabled="loading">Sign up</FormButton>
+    <div
+      class="mt-3 text-xs flex items-center justify-center text-foreground-2 space-x-2"
+    >
+      <!-- 
+        Note the newsletter consent box is here because i got very confused re layout of the panel
+        and didn't figure out a better way to put it where i needed it to be
+       -->
+      <label for="newsletter" class="sxxr-only">
```

Comment:
Why not use `<FormInputCheckbox/>`?

Reply:
i needed to style it a bit more and didn't yet know how to publish a new version of the ui components.